### PR TITLE
Refreshed some metadata classes in preparation for using assemblies as metadata instead of ISDSP

### PR DESCRIPTION
**Pull request by ewoutkramer**

### CODE REVIEW

File: src/Hl7.Fhir.Support.Poco/ElementModel/TypedSerialization.cs [hunk 1]
```diff
@@ -10,53 +10,28 @@
...
-using System.Collections.Concurrent;
 using System.Reflection;
 
 namespace Hl7.Fhir.ElementModel
 {
     public static class TypedSerialization
     {
-        private static readonly ConcurrentDictionary<string, ModelInspector> _inspectedAssemblies = new();
-
-        public static ModelInspector GetInspectorForAssembly(Assembly a)
```
Comment (marcovisserFurore) [inline src/Hl7.Fhir.Support.Poco/ElementModel/TypedSerialization.cs:22]: Do we have here a breaking change? Or was this method not release yet? I am not sure anymore..

**Review by marcovisserFurore — CHANGES_REQUESTED** [review]
See my question

File: src/Hl7.Fhir.Support.Poco/ElementModel/TypedSerialization.cs (hunk 1 above)
Comment (ewoutkramer) [reply src/Hl7.Fhir.Support.Poco/ElementModel/TypedSerialization.cs:22]: We haven't released this code yet. It's just 5 days old!

Comment (marcovisserFurore) [reply src/Hl7.Fhir.Support.Poco/ElementModel/TypedSerialization.cs:22]: Then we can pull it. Thanks

**Review by marcovisserFurore — APPROVED** [review]

**Pull request merged**